.github/composites/docker-build-push/action.yaml

```diff
@@ -144,3 +146,22 @@ runs:
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
         file: "docker/${{ inputs.name }}"
+
+    - name: Install cosign
+      if: "${{ inputs.push == 'true' }}"
+      uses: sigstore/cosign-installer@main
+      with:
+        cosign-release: 'v1.11.0'
+
+    - name: Sign
+      if: "${{ inputs.push == 'true' }}"
+      shell: bash
+      env:
+        COSIGN_EXPERIMENTAL: "true"
+        TAGS: ${{ steps.meta.outputs.tags }}
```

Comment:
what's the echo on this value look like?

Reply:
this should look like `latest,0.9.19.0`